AssettoServer/Network/Tcp/ACTcpClient.cs

```diff
@@ -249,11 +250,18 @@ private async Task ReceiveLoopAsync()
                     if (reader.Buffer.Length == 0)
                         return;
 
-                    byte id = reader.Read<byte>();
-                    if (id != 0x82)
+                    byte byteId = reader.Read<byte>();
+                    if (!Enum.IsDefined(typeof(ACServerProtocol), byteId))
```

Comment:
I think this whole `if` is not necessary. The cast from `byte` to `ACServerProtocol` won't throw any exceptions anyways and from quickly looking over the Enum.IsDefined code it looks like it is allocating memory which adds small but unnecessary GC pressure

Reply:
your right, it just wont match the enum, but it will still do all the if comps afterwards
i guess not allocating is faster